ordered_product/tests.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+from ordered_product.models import Supplier, Client, Product, SupplierProduct, DeliveryLocation, OrderdProduct
+import datetime
+
+
+@pytest.fixture
+def supplier0():
+    return Supplier(user_name="bestSup",
+                    first_name="Yuval",
+                    last_name="Cohen",
+                    password="1234",
+                    user_type="supplier",
+                    business_name="bestSup")
+
+
+@pytest.fixture
+def client0():
+    return Client(user_name="meitar1996",
+                  first_name="Meitar",
+                  last_name="Rizner",
+                  password="123456",
+                  user_type="client",
+                  area="North Israel")
+
+
+@pytest.fixture
+def product0():
+    return Product(qr_code="Q5o76MbdiNbXprNEnHfpcGWFp1CMF8XY",
+                   product_name="Apple",
+                   description="Sweety!")
+
+
+@pytest.fixture
+def supplier_product0():
+    return SupplierProduct(suppler_product_id=1,
+                           qr_code=product0,
+                           user_name=supplier0,
+                           price=10,
+                           quantity=50)
+
+
+@pytest.fixture
+def delivery_location0():
+    return DeliveryLocation(user_name=supplier0,
+                            location="Kiryat Shmona",
+                            date=datetime.date(2022, 12, 30))
+
+
+@pytest.fixture
+def ordered_product0():
```

Comment:
You're creating fixtures for (supplier, client, etc.) but you're not using them. I was expecting to see them being used in this fixture instead of creating yet more instances. You can also save each of them from within those fixtures rather than cluttering this one.

Reply:
I tried this (use fixture as instance) and it didn't work. I will try again

Reply:
Let us know what's not working and we can try and figure this out together.